src/index.html

```diff
@@ -6,9 +6,51 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a class="img" href="#top">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+             class="nav__link"
+             href="#smartphones"
+             data-qa="hover"
+             >
+             Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#gadgets">Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a class="nav__link" href="#tablets">Tablets</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#photo">Photo</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#video">Video</a>
          </li>
```

Let's add spaces between childs